securitycenter/snippets_v2/snippets_findings_v2_test.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from itertools import chain
+import os
+
+import pytest
+
+import snippets_findings_v2
+
+
+@pytest.fixture(scope="module")
+def organization_id():
+    """Get Organization ID from the environment variable"""
+    return os.environ["GCLOUD_ORGANIZATION"]
+
+
+@pytest.fixture(scope="module")
+def source_name(organization_id):
+    from google.cloud import securitycenter_v2
+
+    client = securitycenter_v2.SecurityCenterClient()
+    org_name = f"organizations/{organization_id}"
+
+    source = client.create_source(
+        request={
+            "parent": org_name,
+            "source": {
+                "display_name": "Unit test source",
+                "description": "A new custom source that does X",
+            },
+        }
+    )
+    return source.name
+
+
+
+def test_list_all_findings(organization_id):
+    count = snippets_findings_v2.list_all_findings(organization_id, "-", "global")
```

Comment:
Create the finding before testing to list and group it. You need to have findings for the source.
Also, refactor the tests to verify if the specific finding name is listed and grouped.

Reply:
PTAL at the Java tests for reference.

Reply:
addressed